manager/integration/deploy/test.yaml

```diff
@@ -57,12 +57,24 @@ spec:
       mountPath: /dev
     - name: proc
       mountPath: /host/proc
+    - name: disk-directory
+      mountPath: /tmp/longhorn-test
+      mountPropagation: Bidirectional
+    - name: longhorn
+      mountPath: /var/lib/rancher/longhorn/
+      mountPropagation: Bidirectional
```

Comment:
Are we mounting anything to `/var/lib/rancher/longhorn`?  Not sure about bidirectional mount propagation here.

Reply:
Yeah, I used this default disk to write some file into disk for testing node_controller. Or maybe create a new disk that use `/tmp/longhorn` directory is properly?

Reply:
1. I cannot find the hard disk we additionally mounted inside the container to the `/var/lib/rancher/longhorn`. If that's the case, we don't need mount propagation.
2. Yeah, as you said it's not a great idea to use this host directory for testing. It can potentially leave files in the host filesystem and would be hard to detect that as well. Let's create a new disk for that.